qemu/tests/qemu_img_check_data_integrity.py

```diff
@@ -19,6 +21,7 @@ def run(test, params, env):
     5. If tmp_file_check == no
     5.1 Kill qemu process during writing data in the guest
     6. Boot the guest again, make sure it could boot successfully
+    7. Check the testing image and repair the leaked clusters
```

Comment:
A suggestion, add steps "backup image before testing" and "restore image after testing" into Doc string.